userify/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+# userify/tasks/main.yml
+- name: get userify shim
+  get_url:
+    url: https://shim.userify.com/installer.sh
+    dest: /tmp
+    mode: 0777
+  notify: start_userify
+
+- name: install userify
+  shell: './installer.sh'
+  args:
+    chdir: /tmp
+    creates: /opt/userify
+  environment:
+    api_id: "{{userify_api_id}}"
```

Comment:
If we're going to use variables, we should have defaults definted in `<role>/defaults/main.yml` (even if they're just empty strings or something) as documentation of the possible variables we can override